### DISCUSSION TITLE
I don't get the blue button to compile

### DISCUSSION THREAD

**Discussion by Drbacteria2** [nickcoutsos/keymap-editor#271] [Q&A]
Good.
I am using keymap-editor for a keyboard with a Cyber60 pcb, when I use the repository to create my own keymap and save the changes, I don't get the blue firmware download button.
I would like to know what I am doing wrong in this regard and what I need to modify to get my keymap compiled.
Thanks a lot and great job

**Comment by nickcoutsos** [comment]
The button wouldn't show if there's no build to link to. I think you need to enable actions on your repository. If you go to the actions tab (https://github.com/Drbacteria2/zmk-config-4pplet/actions) you may see a button to enable actions, otherwise you may need to look around in your repository settings somewhere -- I can't get into specifics at the moment but if you're still having trouble let me know and I'll have a closer look.

**Comment by Drbacteria2** [comment]
Thank you very much for your reply and your help.
The problem was indeed that I had not enabled the actions in the repository that I had cloned and that the designer of the board sent me. 
now it works fine, it compiles and creates the uf2 file without problem.
I leave this here in case someone has the same problem as me.
